frontend/src/ui-components/components/tabs/Tabs.vue

```diff
@@ -1,30 +1,92 @@
-            <router-link
-                v-for="(tab, $index) in scopedTabs"
-                :key="tab.label"
-                data-el="ff-tab"
-                class="ff-tab-option transition-fade--color"
-                :to="tab.to"
-                :data-nav="tab.tag"
-                @click="selectTab($index)"
-            >
-                {{ tab.label }}
-                <span v-if="tab.featureUnavailable" v-ff-tooltip="'Not available in this Team Tier'" data-el="premium-feature">
-                    <SparklesIcon class="ff-icon transition-fade--color hollow" style="stroke-width: 1;" />
-                </span>
-            </router-link>
-        </ul>
+    <div class="ff-tabs-wrapper" :class="{ 'ff-tabs-wrapper--overflow-enabled': enableOverflow }">
+        <HeadlessMenu v-if="enableOverflow && hasHiddenLeft" as="div" class="ff-tabs__menu-wrapper ff-tabs__menu-wrapper--left">
+            <MenuButton class="ff-tabs__overflow-button ff-tabs__overflow-button--left" aria-label="Show hidden tabs on the left">
+                <ChevronLeftIcon class="ff-icon" />
+                <ChevronLeftIcon class="ff-icon ff-icon-second" />
+            </MenuButton>
+            <MenuItems class="z-50 absolute left-0 top-full origin-top-left w-56 mt-1 bg-white divide-y divide-gray-100 rounded overflow-hidden shadow-lg ring-1 ring-black ring-opacity-10 focus:outline-none">
+                <MenuItem
+                    v-for="tab in hiddenLeftTabs"
+                    :key="'left-' + tab.label"
+                    v-slot="{ active }"
+                >
+                    <button
+                        :class="[active ? 'bg-gray-200' : '', 'block w-full text-left px-4 py-2 text-sm text-gray-700']"
+                        @click="onDropdownItemClick(tab)"
+                    >
+                        <img v-if="tab.icon" :src="tab.icon" class="h-4 v-4 inline rounded mr-1" alt="">
+                        {{ tab.label }}
+                    </button>
+                </MenuItem>
+            </MenuItems>
+        </HeadlessMenu>
+
+        <div
+            ref="scrollContainer"
+            class="ff-tabs__scroll-container"
+            :class="{ 'ff-tabs__scroll-container--overflow': enableOverflow }"
+            @scroll="handleScroll"
+        >
+            <ul class="ff-tabs" :class="'ff-tabs--' + orientation">
+                <router-link
+                    v-for="(tab, $index) in scopedTabs"
+                    :key="tab.label"
+                    :ref="'tab-' + $index"
+                    data-el="ff-tab"
+                    class="ff-tab-option transition-fade--color"
+                    :to="tab.to"
+                    :data-nav="tab.tag"
+                    @click="selectTab($index)"
+                >
+                    <img v-if="tab.icon" :src="tab.icon" class="ff-tab-icon" alt="">
+                    <span class="ff-tab-label">{{ tab.label }}</span>
```

Comment:
should be <component :is="tab.icon"" ...

Reply:
Fixed with https://github.com/FlowFuse/flowfuse/commit/f617660d6